spec/system/users_spec.rb

```diff
@@ -52,6 +52,16 @@
         expect(current_path).to eq(root_path)
       end
 
+      it 'saves the last_success_otp_at' do
```

Comment:
WHEN or WHY does it save the last_success_otp_at?
Is it when the user successfully otps?